tests/security/pam/pam_su.pm

```diff
@@ -23,14 +24,25 @@ sub run {
     validate_script_output "id $user | grep $group || echo 'check pass'", sub { m/check pass/ };
 
     # Modify the PAM configuration files
-    my $su_file = '/etc/pam.d/su';
+    my $su_file = '';
+    my $sul_file = '';
+    my $su_file_tw = '';
+    my $sul_file_tw = '';
+    if (is_sle || is_leap) {
+        $su_file = '/etc/pam.d/su';
+        $sul_file = '/etc/pam.d/su-l';
+        $su_file_tw = '/usr/etc/pam.d/su';
+        $sul_file_tw = '/usr/etc/pam.d/su-l';
+    } else {
+        $su_file = '/lib/pam.d/su';
```

Comment:
same issue here - we modify direcly in lib.

the copy job does not fail as /lib == /usr/lib, so the check further down if the file exists returns true.

The admin modified files are to remain in /etc/pam.d (so copy from /usr/lib/pam.d to /etc/pam.d)